aten/src/ATen/native/quantized/affine_quantizer.cpp

```diff
@@ -314,6 +386,25 @@ CAFFE2_API float dequantize_val(double scale, int64_t zero_point, T value) {
 }
 #endif // USE_FBGEMM
 
+/*
+* Quantize value based on the following equation
+* Qx = (Xf - bias) * inv_scale
```

Comment:
just to clarify, does `bias = -1 * zero_point * scale` (to get `Qx = round(Xf/scale - zero_point)`?

if yes, maybe we can make the API clearer by having the callsites also use the "bias" name?

Reply:
I think this was a little confusing as I was trying to match the C2 equation  `(Xf-bias) * inv_scale` in numerics by setting zero_point = bias.
I had a chat with Raghu about this and we thought it would be better to use zero_point = (-bias/scale) for this case so that we can use similar quantize equation to what we have now. 
So the current equation: `Xq = Xf * inv_scale + zp`
Substituting zero_point
`Xq = Xf * inv_scale + (-bias *inv_scale)`
`Xq = (Xf - bias) * inv_scale`, which is the same as what caffe2 uses for embedding layers today. There may be some numerical differences due to the division rounding.
I'll update the PR with the change for this.

Reply:
yes, thanks, that matches what I assumed from reading the code.  I think that works well, but maybe we can have the callers use "bias" and not "zero_point" as the argument name then, to prevent confusion?  i.e.

```
quantize_val_float_qparams<qint8>(float scale, float bias, float value);
# instead of
quantize_val_float_qparams<qint8>(float scale, float zero_point, float value);
```